### PR TITLE
[SW-1854] Expose Individual Chunks via REST API

### PR DESCRIPTION
**Pull request by mn-mikke**

### CODE REVIEW

File: extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/SerializationUtils.scala [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+  def sendTimestamp(ab: AutoBuffer, time: Long): Unit = {
+    sendLong(ab, time)
+  }
+
+  def sendTimestamp(ab: AutoBuffer, data: Timestamp): Unit = {
+    sendLong(ab, data.getTime)
+  }
+
+  private[serde] def sendNA(ab: AutoBuffer, expectedType: Byte): Unit = expectedType match {
```
Comment (jakubhava) [inline extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/SerializationUtils.scala]: Maybe typo? private[serde]

**Comment by jakubhava** [comment]
I'm excited that we will have full control over external backend in SW repo 💯

**Comment by jakubhava** [comment]
Note: We also need to automatically add the jar to `-libjars` in case of automatic mode of external backend and mention this in the documentation

File: extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/SerializationUtils.scala (hunk 1 above)
Comment (mn-mikke) [reply extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/SerializationUtils.scala]: Would you like me to rename `serde` package?

Comment (jakubhava) [reply extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/SerializationUtils.scala]: Maybe something a little bit more explanatory would help? ( Unless this is a know term which just I don't know, in that case it's ok :D )

**Pull request merged**